R/results_fig_tabs.R

```diff
@@ -483,3 +517,25 @@ language_sas_row <- results %>%
   group_by(language_sas) %>% 
   mutate(tot = sum(n), per = n/tot*100) %>% 
   filter(available_code) 
+
+
+# 5. POST PEER REVIEW: REQUESTED TABLES ----------------------------------------------
+
+# each individual language listed 
+language_all <- results %>% 
+  select(language) %>%
+  # put each instance into a separate row and count them
+  separate_rows(language, sep = ",") %>% 
+  separate_rows(language, sep = "\\band\\b") %>% 
+  # remove leading blanks 
+  mutate(language = trimws(language)) %>% 
```

Comment:
`trimws` is new for me - v useful!

Reply:
Google & stack exchange deserve all the credit..!